src/Opg/Lpa/DataModel/Lpa/Payment/Calculator.php

```diff
@@ -5,61 +5,53 @@
 
 class Calculator
 {
-    const STANDARD_FEE = 110;
+    const STANDARD_FEE = 82;
 
     /**
      * Calculate LPA payment amount
      *
      * @param Lpa $lpa
      * @return NULL|Payment
      */
-    static public function calculate(Lpa $lpa)
+    public static function calculate(Lpa $lpa)
     {
-        if(!($lpa->payment instanceof Payment)) return null;
-
-        $isRepeatApplication = ($lpa->repeatCaseNumber != null);
-
-        if(($lpa->payment->reducedFeeReceivesBenefits) && ($lpa->payment->reducedFeeAwardedDamages)) {
+        if (!$lpa->payment instanceof Payment) {
+            return null;
+        }
 
+        if ($lpa->payment->reducedFeeReceivesBenefits && $lpa->payment->reducedFeeAwardedDamages) {
             $amount = self::getBenefitsFee();
-
         } else {
+            $isRepeatApplication = ($lpa->repeatCaseNumber != null);
```

Comment:
!= would give the unexpected results if the var is zero or false or empty string


$isRepeatApplication will